tests/integration/analytics/test_analytics_collectors.py

```diff
@@ -424,6 +422,52 @@ def test_event_streams_table_collector(
             assert lines[0][3] == default_event_stream.event_stream_type
 
 
+@pytest.mark.django_db
+def test_event_streams_table_by_activation_collector(
+    default_activation: models.Activation,
+    new_activation: models.Activation,
+    default_event_streams: list[models.EventStream],
+):
+    default_activation.event_streams.add(default_event_streams[0])
+    new_activation.event_streams.add(default_event_streams[1])
+
+    until = now()
+    time_start = until - timedelta(hours=9)
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        collectors.event_streams_by_activation_table(
+            time_start, tmpdir, until=now() + timedelta(seconds=1)
+        )
+        with open(
+            os.path.join(tmpdir, "event_streams_by_activation_table.csv")
+        ) as f:
+            reader = csv.reader(f)
+
+            header = next(reader)
+            lines = list(reader)
+
+            assert header == [
+                "activation_id",
+                "event_stream_id",
+                "name",
+                "event_stream_type",
+                "eda_credential_id",
+                "events_received",
+                "last_event_received_at",
+                "organization_id",
```

Comment:
Maybe we should do the inverse in this test, and ensure `name` is not collected here

Reply:
same as above.